ch_tools/monrun_checks/ch_replication_lag.py

```diff
@@ -56,6 +56,10 @@
 )
 @click.pass_context
 def replication_lag_command(ctx, xcrit, crit, warn, mwarn, mcrit, verbose):
+    return estimate_replication_lag(ctx, xcrit, crit, warn, mwarn, mcrit, verbose)
+
+
+def estimate_replication_lag(ctx, xcrit=3600, crit=6000, warn=300, mwarn=50.0, mcrit=90.0, verbose=0):
```

Comment:
Lets move hardcoded values to constants in top of file. They might be mismatched. And it seems already are (`crit = 600` above).